api/db/repositories/verification/verification.js

```diff
@@ -12,22 +13,37 @@ class Verification {
     static async IdExists (resource, requestedId) {
         switch (resource) {
             case Resources.Customers:
-                const requestedCustomer = await Customers.findById(requestedId);
+                const requestedCustomer = await Customers.findById(requestedId)
 
-                this.ResourceIsNull(requestedCustomer);
-                break;
+                this.ResourceIsNull(Resources.Customers, requestedCustomer)
+                break
             
             case Resources.Products:
-                const requestedProduct = await Products.findById(requestedId);
+                const requestedProduct = await Products.findById(requestedId)
 
-                this.ResourceIsNull(requestedProduct);
-                break;
+                this.ResourceIsNull(Resources.Products, requestedProduct)
+                break
+
+            case Resources.Orders:
+                const requestedOrder = await Orders.findById(requestedId)
+
+                this.ResourceIsNull(Resources.Order, requestedOrder)
+                break
         }
     }
 
-    static ResourceIsNull(requestedItem) {
+    static ResourceIsNull(resources, requestedItem) {
```

Comment:
lower case

Reply:
shouldn't it be lower case?

Reply:
isn't that what I said? :D

Reply:
aaaaaa, methods. right

Reply:
and also please rename `resources` parameter to `resource` because it's singular not plural